common/tools/audio.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+from sisyphus import tk
+
+from i6_core.tools.git import CloneGitRepositoryJob
+from i6_core.tools.compile import MakeJob
+
+
+def compile_ffmpeg_binary(
+    branch: Optional[str] = None, commit: Optional[str] = None
+) -> tk.Path:
+    """
+    Compiles FFmpeg using the official Github master.
+
+    For requirements see:
+    https://trac.ffmpeg.org/wiki/CompilationGuide/Ubuntu
+
+    The current configure options are set in a way that the resulting FFmpeg binary lacks many features.
+    This is done because we only need limited codec capabilities.
+
+    :param branch: specify a specific branch or tag, e.g. "n4.1.4"
+    :param commit: specify a specific commit
+    :return: Path object pointing to the "ffmpeg" binary executable
+    """
+    ffmpeg_repo = CloneGitRepositoryJob(
+        "https://github.com/FFmpeg/FFmpeg",
+        branch=branch,
+        commit=commit,
+    )
+
+    ffmpeg_make_binary = MakeJob(
+        folder=ffmpeg_repo.out_repository,
+        make_sequence=None,
```

Comment:
I think `None` defaults to `["all"]` so I suggest to explicitly write that to make it more clear what make command is executed.
```suggestion
        make_sequence=["all"],
```